app/presenters/routine_flow_presenter.rb

```diff
@@ -1,14 +1,26 @@
-class RoutineFlowPresenter
-  include ViewCompatibilityPack
-
+class RoutineFlowPresenter < BasePresenter
+  presents :routine_flow
   delegate :routine, to: :routine_flow, prefix: false
 
-  def initialize(routine_flow)
-    @routine_flow = routine_flow
+  def title
+    routine_flow.title
   end
 
-  def routine_flow_title
-    routine_flow.title
+  def completed_at
+    formatted_date_time = routine_flow.completed_at.to_s(:long_ordinal)
+    "completed on #{formatted_date_time}"
+  end
+
+  def reports_button
+    link_to 'Reports',
+            routine_routine_flow_reports_path(routine),
+            class: 'button is-large is-info is-fullwidth'
+  end
+
+  def routines_button
+    link_to 'Routines',
+            routines_path,
+            class: 'button is-large is-info is-fullwidth'
```

Comment:
Are these presenting routine_flow? Where should we move so it semantically makes more sense? 🤔